core/arch/arm/plat-rcar/main.c

```diff
@@ -76,3 +82,25 @@ unsigned long plat_get_aslr_seed(void)
 
 	return seed;
 }
+void main_init_gic(void)
+{
+	vaddr_t gicc_base;
+	vaddr_t gicd_base;
+
+	gicc_base = core_mmu_get_va(GICC_BASE, MEM_AREA_IO_SEC, GIC_CPU_REG_SIZE);
+	gicd_base = core_mmu_get_va(GICD_BASE, MEM_AREA_IO_SEC, GIC_DIST_REG_SIZE);
+
+	if (!gicc_base || !gicd_base)
+		panic();
+
+	/* Initialize GIC */
+	gic_init(&gic_data, gicc_base, gicd_base);
+	itr_init(&gic_data.chip);
+}
+
+#if CFG_TEE_CORE_NB_CORE > 1
```

Comment:
I don't suppose this is needed. The amount of memory saved must be minimal. Finding out a way of excluding `reset_secondary()` should enable saving a few more bytes.

Reply:
No worries. When it's needed, it's needed.